### ISSUE TITLE
[Accessibility] Non-descriptive error message

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#322]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-422) to this repo.

- Audit ID: WooExt-517
- Component/Page: Checkout
- Affected Element: There was a problem with your payment option.
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/checkout/

## WCAG Compliance

- Success Criterion: 3.3.1 Error Identification
- WCAG Version: 2
- Level: A
- Severity: High

## Issue Description

The error message lacks meaningful information, failing to specify the nature of the issue, the location of the error, or provide clear instructions on how to resolve it.

### Impacted Populations

All people including those who do not identify as disabled

## Recommended Fix

Provide a summary in the error message to indicate the input fields containing the errors and, if possible, instructions on how to fix it or the type of input that is required from the user. The error message should help the user resolve the issue effectively. For example: "There was an issue with your payment option. Please sign in to Amazon to continue".

## Code Example
```
<div class="wc-block-store-notice wc-block-components-notice-banner is-error is-dismissible"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M12 3.2c-4.8 0-8.8 3.9-8.8 8.8 0 4.8 3.9 8.8 8.8 8.8 4.8 0 8.8-3.9 8.8-8.8 0-4.8-4-8.8-8.8-8.8zm0 16c-4 0-7.2-3.3-7.2-7.2C4.8 8 8 4.8 12 4.8s7.2 3.3 7.2 7.2c0 4-3.2 7.2-7.2 7.2zM11 17h2v-6h-2v6zm0-8h2V7h-2v2z"></path></svg><div class="wc-block-components-notice-banner__content"><div>There was a problem with your payment option.</div></div><button class="wc-block-components-button wp-element-button wc-block-components-notice-banner__dismiss contained" aria-label="Dismiss this notice" type="button"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"></path></svg></button></div>
```
## Screenshots

https://drive.google.com/file/d/1lVwL4i27svrr5zAx5uqgPwoBE5oim-3v/view?usp=drivesdk

**Comment by ChristianAF** [comment]
Hey @amberhinds, this issue is not valid for the plugin. It's related to the Amazon SDK. We can not handle the Amazon Pay rendering styling

**Comment by amberhinds** [comment]
Hi @ChristianAF, is there a way to report this upstream?

**Comment by ChristianAF** [comment]
Sorry, this one is on us
